### PR TITLE
WIP: Adds Timezone to GUI.

### PR DESCRIPTION
**Pull request by raghunayyar**
@georgehrke you can move ahead with this now. Please commit all timezone changes you want to push here. :)

should take care of atleast #51

### CODE REVIEW

File: templates/part.eventsinfo.php [hunk 1]
```diff
@@ -25,49 +25,66 @@
...
+
+
+<fieldset ng-show="fromdatetimezone" class="event-fieldset">
+	<label class="label"><?php p($l->t('Event Start')); ?></label>
+	<select ng-options="timezone for timezone in timezones" ng-model="properties.dstart.tzid"></select>
+</fieldset>
+
+<fieldset ng-show="todatetimezone" class="event-fieldset">
+	<label class="label"><?php p($l->t('Event End')); ?></label>
+	<select ng-options="timezone for timezone in timezones" ng-model="currenttotimezone.dtend.tzid"></select>
```
Comment (georgehrke) [inline templates/part.eventsinfo.php]: the ng-model should be `properties.dtend.tzid`

File: templates/part.eventsinfo.php [hunk 2]
```diff
@@ -25,49 +25,66 @@
...
+		  ng-model="properties.allDay"
+		  id="alldayeventcheckbox" class="event-checkbox" />
+		<label for="alldayeventcheckbox"><?php p($l->t('All day Event'))?></label>
+	</div>
+</fieldset>
+
+
+<fieldset ng-show="fromdatetimezone" class="event-fieldset">
+	<label class="label"><?php p($l->t('Event Start')); ?></label>
+	<select ng-options="timezone for timezone in timezones" ng-model="properties.dstart.tzid"></select>
```
Comment (georgehrke) [inline templates/part.eventsinfo.php]: ng-model should be `properties.dtstart.tzid`

**Comment by raghunayyar** [comment]
@georgehrke should we get this in? I will rebase it if you want me to. Or, you want to work on timezone JS?

**Comment by georgehrke** [comment]
please rebase

**Comment by georgehrke** [comment]
let's implement this in https://github.com/owncloud/calendar-rework/pull/113

**Pull request closed without merging**